updated BVOC documentation (#3909)
ESCOMP/CTSM: doc/source/tech_note/BVOCs/CLM50_Tech_Note_BVOCs.rst

Biogenic Volatile Organic Compounds (BVOCs)
===============================================

> This section briefly describes the biogenic volatile organic compound (BVOC) emissions model implemented in CLM. The CLM3 version (Levis et al. 2003; Oleson et al. 2004) was based on Guenther et al. (1995). Heald et al. (2008) updated this scheme in CLM4 based on Guenther et al (2006). The current version was first implemented in CLM4.5 and is based on MEGAN2.1 discussed in detail in Guenther et al. (2012). CLM-MEGAN now includes these features: 1) expansion to 147 chemical compounds, 2) the treatment of the light-dependent fraction (LDF) for each compound, 3) inclusion of the inhibition of isoprene emission by atmospheric CO\ :sub:`2`, 4) emission factors mapped to the specific PFTs of the CLM, 5) the impact of drought, and 6) high-latitude specific isoprene emissions.

Reviewer: ```suggestion
This section briefly describes the biogenic volatile organic compound (BVOC) emissions model implemented in CLM. The CLM3 version (:ref:`Levis et al. 2003 <Levisetal2003>`; :ref:`Oleson et al. 2004 <Olesonetal2004>`) was based on :ref:`Guenther et al. (1995) <Guentheretal1995>`. :ref:`Heald et al. (2008) <Healdetal2008>` updated this scheme in CLM4 based on :ref:`Guenther et al (2006) <Guentheretal2006>`. The current version was first implemented in CLM4.5 and is currently based on MEGAN2.1 discussed in detail in :ref:`Guenther et al. (2012) <Guentheretal2012>`. As of CLM5, CLM-MEGAN has included these features: 1) expansion to 147 chemical compounds, 2) the treatment of the light-dependent fraction (LDF) for each compound, 3) inclusion of the inhibition of isoprene emission by atmospheric CO\ :sub:`2`, 4) emission factors mapped to the specific PFTs of the CLM. As of CLM6, CLM-MEGAN includes two new features: 5) the impact of drought, and 6) high-latitude specific isoprene emissions.
```

> MEGAN2.1 now describes the emissions of speciated monoterpenes, sesquiterpenes, oxygenated VOCs as well as isoprene. A flexible scheme has been implemented in the CLM to specify a subset of emissions. This allows for additional flexibility in grouping chemical compounds to form the lumped species frequently used in atmospheric chemistry. The mapping or grouping is therefore defined through a namelist parameter in drv\_flds\_in, e.g. megan\_specifier = 'ISOP = isoprene', 'BIGALK pentane + hexane + heptane + tricyclene'.

Reviewer: ```suggestion
MEGAN2.1 describes the emissions of speciated monoterpenes, sesquiterpenes, oxygenated VOCs as well as isoprene. A flexible scheme has been implemented in the CLM to specify a subset of emissions. This allows for additional flexibility in grouping chemical compounds to form the lumped species frequently used in atmospheric chemistry. The mapping or grouping is therefore defined through a namelist parameter in drv\_flds\_in, e.g. megan\_specifier = 'ISOP = isoprene', 'BIGALK pentane + hexane + heptane + tricyclene'.
```

Terrestrial BVOC emissions from plants to the atmosphere are expressed as a flux, :math:`F_{i}` (:math:`\mu` \ g C m\ :sup:`-2` ground area h\ :sup:`-1`), for emission of chemical compound :math:`i`

.. math::
   :label: flux equation

   F_{i} =\gamma _{i} \rho \sum _{j}\varepsilon _{i,j}  \left(wt\right)_{j}

> where :math:`\gamma _{i}` is the emission activity factor accounting for responses to meteorological and phenological conditions, :math:`\rho` is the canopy loss and production factor also known as escape efficiency (set to 1), and :math:`\varepsilon _{i,\, j}` (:math:`\mu` \ g C m\ :sup:`-2` ground area h\ :sup:`-1`) is the emission factor at standard conditions of light, temperature, and leaf area for plant functional type *j* with fractional coverage :math:`\left(wt\right)_{j}` (Guenther et al. 2012). The emission activity factor :math:`\gamma _{i}` depends on plant functional type, temperature, LAI, leaf age, and soil moisture (Guenther et al. 2012) For isoprene only, the effect of CO\ :sub:`2` inhibition is now included as described by Heald et al. (2009). Previously, only isoprene was treated as a light-dependent emission. In MEGAN2.1, each chemical compound is assigned a LDF (ranging from 1.0 for isoprene to 0.2 for some monoterpenes, VOCs and acetone). The activity factor for the light response of emissions is therefore estimated as:

Reviewer: ```suggestion
where :math:`\gamma _{i}` is the emission activity factor accounting for responses to meteorological and phenological conditions, :math:`\rho` is the canopy loss and production factor also known as escape efficiency (set to 1), and :math:`\varepsilon _{i,\, j}` (:math:`\mu` \ g C m\ :sup:`-2` ground area h\ :sup:`-1`) is the emission factor at standard conditions of light, temperature, and leaf area for plant functional type *j* with fractional coverage :math:`\left(wt\right)_{j}` (Guenther et al. 2012). The emission activity factor :math:`\gamma _{i}` depends on plant functional type, temperature, LAI, leaf age, and soil moisture (Guenther et al. 2012) For isoprene only, the effect of CO\ :sub:`2` inhibition is now included as described by :ref:`Heald et al. (2009) <Healdetal2009>`. Previously, only isoprene was treated as a light-dependent emission. In MEGAN2.1, each chemical compound is assigned a LDF (ranging from 1.0 for isoprene to 0.2 for some monoterpenes, VOCs and acetone). The activity factor for the light response of emissions is therefore estimated as:
```

.. math::
   :label: light-dependent activity factor

   \gamma _{P,\, i} =\left(1-LDF_{i} \right)+\gamma _{P\_ LDF} LDF_{i}

where the LDF activity factor (:math:`\gamma _{P\_ LDF}` ) is specified as a function of PAR as in previous versions of MEGAN.

The values for each emission factor :math:`\epsilon _{i,\, j}` are now available for each of the plant functional types in the CLM and each chemical compound. This information is provided in an external file, allowing for more frequent and easier updates.

> The impact of drought on isoprene emissions is based on the theory proposed by Potosnak et al. (2014). Specifically, isoprene emissions are expected to increase under mild to moderate drought because drought raises leaf temperature, which stimulates isoprene emissions. Under severe drought, however, isoprene emissions are inhibited because substrate supply becomes constrained. Because the effect of leaf temperature is already represented by the leaf temperature activity factor :math:`\gamma _{T}` and its influence on isoprene emissions, only the inhibitory effect of severe drought (substrate supply impact, :math:`\gamma _{sub}` ) is parameterized as:

Reviewer: ```suggestion
The impact of drought on isoprene emissions is based on the theory proposed by :ref:`Potosnak et al. (2014) <Potosnaketal2014>`. Specifically, isoprene emissions are expected to increase under mild to moderate drought because drought raises leaf temperature, which stimulates isoprene emissions. Under severe drought, however, isoprene emissions are inhibited because substrate supply becomes constrained. Because the effect of leaf temperature is already represented by the leaf temperature activity factor :math:`\gamma _{T}` and its influence on isoprene emissions, only the inhibitory effect of severe drought (substrate supply impact, :math:`\gamma _{sub}` ) is parameterized as:
```

.. math::
   :label: drought factor

   \gamma _{sub} =\frac{1}{1+b_{1} e^{a1 (\beta -0.2)}}

> where :math:`a1=-7.45` and :math:`b1=3.26` are empirical parameters (described in Wang et al., 2022).

Reviewer: ```suggestion
where :math:`a_1=-7.4463` and :math:`b_1=3.2552` are empirical parameters (described in :ref:`Wang et al., 2022 <Wangetal2022>`).
```

> Compared with Guenther et al. (2012), updates have been made to represent isoprene emissions from high-latitude plants, specifically boreal broadleaf deciduous shrubs (BBDS) and C3 Arctic grass (C3AG), in order to account for acclimation processes. These updates are based on leaf-enclosure and in situ measurements conducted at Toolik Field Station in Alaska, USA (Wang et al., 2024a, 2024b).

Reviewer: ```suggestion
Compared with Guenther et al. (2012), updates have been made to represent isoprene emissions from high-latitude plants, specifically boreal broadleaf deciduous shrubs (BBDS) and C3 Arctic grass (C3AG), in order to account for acclimation processes. These updates are based on leaf-enclosure and in situ measurements conducted at Toolik Field Station in Alaska, USA (:ref:Wang et al., 2024a, 2024b <Wangetal2024a>`).
```

For BBDS, the isoprene emission factor is adjusted according to the mean temperature of the previous day as:

.. math::
   :label: boreal shrub adjustment factor

   E_{opt\_bbds} = 7.9 e^{0.22 (T_{24}-297.15)}

where :math:`T_{24}` denotes the mean air temperature of the preceding day (Wang et al., 2024a).
For C3AG, the isoprene emission factor responds over a longer timescale of 10 days (Wang et al., 2024b) and is parameterized as a function of the mean air temperature over the preceding 10 days (:math:`T_{240}`):

.. math::
   :label: C3 arctic grass adjustment factor

>    E_{opt\_C3AG} = e^{0.22 (T_{240}-288.15)}

Reviewer: ```suggestion
   E_{opt\_C3AG} = e^{0.217 (T_{240}-288.15)}
```

Reviewer: Here's another constant that I could not find in the code. Did I pick the correct value from the code?

Author: Yes, that looks correct.

Reviewer: Ok, from another look at the code...
- 0.217 looks correct for the 0.22 used for BBDS, while
- for C3AG the coefficient in the code is 0.12 if I'm not mistaken

I will push these changes in the next few minutes.

In addition, a dynamic temperature response curve for C3AG depends on recent temperature history as:

.. math::
   :label: C3 arctic grass leaf temperature factor

>    \gamma_{T\_C3AG} = E_{opt\_C3AG} e^{(C_{C3AG}/R (1/303.15 - 1/T_{leaf}))}

Reviewer: This formula is a bit difficult to read the way that it renders. You can download to your laptop my copy of this PR's built documentation `/glade/derecho/scratch/slevis/preview_docs_pr.ESCOMP.CTSM.pr-3909/ESCOMP-CTSM-a074bdb/doc/_build/html` and then click on index.html to view. Here are my concerns:
- [x] Pls confirm that the formula agrees with what's in the code
- [x] See if you can format it to make easier to read
- [x] Has the constant R been defined?

- [x] Looking at this formula made me notice that several param values in this doc file didn't match the code. I have pointed out that ones that I caught. Pls confirm that I replaced them with the correct values because I mostly guessed. Also pls check that I didn't miss any. To explain, we prefer to report exact param values in the documentation to make it easier to follow the documentation alongside the code, as well as to facilitate the ability to reproduce results if one were to rewrite the model starting from the documentation.

Reviewer: @lkemmons pls ping me here in the PR when you feel the rest can be resolved, and I will approve the PR and merge to the b4b-dev branch.

Author: Checking with @HuiWangWanderInGitHub to verify the gamma_T equation.

Author: @slevis-lmwg  Hui confirms the documentation matches the code.  I made some updates to [doc/source/tech_note/BVOCs/CLM50_Tech_Note_BVOCs.rst](https://github.com/lkemmons/CTSM/blob/lkemmons-docs-bvocs/doc/source/tech_note/BVOCs/CLM50_Tech_Note_BVOCs.rst) 
that might be clearer.

Reviewer: Thank you. I also rearranged the "/ R" using the code as my guide. The equation ends up the same. I just find it easier to interpret this way. I will push the change in a moment.

> where :math:`T_{leaf}` denotes the leaf temperature and :math:`C_{C3AG}` is the parameter controlling the isoprene temperature response of C3AG and changes varies with :math `T_{240}` as:

Reviewer: ```suggestion
where :math:`T_{leaf}` denotes the leaf temperature and :math:`C_{C3AG}` is the parameter controlling the isoprene temperature response of C3AG and changes varies with :math:`T_{240}` as:
```

.. math::
   :label: C3 arctic grass parameter

>    C_{C3AG} = 95 + 9.5 e^{0.53 (288.15-T_{240})}

Reviewer: ```suggestion
   C_{C3AG} = 95 + 9.49 e^{0.53 (288.15-T_{240})}
```

Reviewer: Ok, done committing the "obvious" ones. Here's one of the constants that I could not find in the code, but that had a likely counterpart. Did I pick the correct value?

> Compared with Guenther et al. (2012), updates have been made to represent isoprene emissions from high-latitude plants, specifically boreal broadleaf deciduous shrubs (BBDS) and C3 Arctic grass (C3AG), in order to account for acclimation processes. These updates are based on leaf-enclosure and in situ measurements conducted at Toolik Field Station in Alaska, USA (:ref:Wang et al., 2024a, 2024b <Wangetal2024a>`).

Reviewer: ```suggestion
Compared with Guenther et al. (2012), updates have been made to represent isoprene emissions from high-latitude plants, specifically boreal broadleaf deciduous shrubs (BBDS) and C3 Arctic grass (C3AG), in order to account for acclimation processes. These updates are based on leaf-enclosure and in situ measurements conducted at Toolik Field Station in Alaska, USA (:ref:`Wang et al., 2024a, 2024b <Wangetal2024a>`).
```
